tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge/signup_kme.cpp

```diff
@@ -130,10 +130,53 @@ tcf_err_t tcf::enclave_api::enclave_data_kme::CreateEnclaveDataKME(
     }
 
     return result;
-}  // tcf::enclave_api::enclave_data_kme::CreateEnclaveDataKME
+}  // SignupDataKME::CreateEnclaveData
+
+// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+tcf_err_t SignupDataKME::UnsealEnclaveData(
+    StringArray& outPublicEnclaveData) {
+    tcf_err_t result = TCF_SUCCESS;
+
+    try {
+        outPublicEnclaveData.resize(SignupData::CalculatePublicEnclaveDataSize());
```

Comment:
indentation

Reply:
I think indentation is proper. Are you pointing at line more than 80 chars? I will correct it.